graphics/vector3_for_2d_rendering.py

```diff
@@ -0,0 +1,62 @@
+"""
+render 3d points for 2d surfaces.
+"""
+
+import math
+
+__version__ = "2020.9.26"
+__author__ = "xcodz-dot"
+
+
+def convert_to_2d(x: float, y: float, z: float, scale: float,
+                  distance: float) -> (float, float):
+    """
+    Converts 3d point to a 2d drawable point
+
+    >>> convert_to_2d(1.0, 2.0, 3.0, 10.0, 10.0)
+    (7.6923076923076925, 15.384615384615385)
+    """
+    if not all(isinstance(val, (float, int)) for val in locals().values()):
+        raise ValueError(f"Input values must either be float or int: {list(locals().values())}")
+    projected_x = ((x * distance) / (z + distance)) * scale
+    projected_y = ((y * distance) / (z + distance)) * scale
+    return projected_x, projected_y
+
+
+def rotate(x: float, y: float, z: float, axis: str,
+           angle: float) -> (float, float, float):
+    """
+    rotate a point around a certain axis with a certain angle
+    angle can be any integer between 1, 360 and axis can be any one of
+    'x', 'y', 'z'
+
+    >>> rotate(1.0, 2.0, 3.0, 'y', 90.0)
+    (3.130524675073759, 2.0, 0.4470070007889556)
+    """
+    if not isinstance(angle, (int, float)):
+        raise ValueError("Angel must be int or float")
+    if angle > 360 or angle < 0:
+        raise ValueError("Angle is supposed to be in between 0, 360")
+    if not isinstance(x, (int, float)):
+        raise ValueError("x must be int or float")
+    if not isinstance(y, (int, float)):
+        raise ValueError("y must be int or float")
+    if not isinstance(z, (int, float)):
+        raise ValueError("z must be int or float")
```

Comment:
The same thing can be done here except for the angle checking part. 😁

Reply:
ok, doing

Reply:
Also, this is better
```python
not 0 <= angle <= 360
```

Reply:
sorry, but it will take time, (I am a school student, I have to attend zoom meeting). Please wait for 40 mins

Reply:
Try to run all the checks mentioned in [.travis.yml](https://github.com/TheAlgorithms/Python/blob/master/.travis.yml) before committing the changes, so that you know if there's going to be any small problem 😁

Reply:
No problem, take your time.

Reply:
done, waiting for travis to approve my code